### DISCUSSION TITLE
Disable pop-up Unhandled Exception in script error

### DISCUSSION THREAD

**Discussion by Dariusz1989** [pyinstaller/pyinstaller#6292] [Help]
Hello
How can I disable that error? I don't want any pop-ups with stack trace logs/etc..
![image](https://user-images.githubusercontent.com/13185728/136781770-9f60e6c9-5577-4f02-811d-e1076298d8a6.png)
If anything, I would want it to save to a specific file in a folder I can specify & encryption so I can debug it latel.
Regards
Dariusz

**Comment by bwoodsend** [comment]
With the [ --disable-windowed-traceback](https://pyinstaller.readthedocs.io/en/stable/usage.html#cmdoption-disable-windowed-traceback) flag. But really the correct solution is to implement your own exception handling which catches the error and logs it somewhere - otherwise you're completely stuffed if your users ever have a platform dependent issue which you can't reproduce on your own machine.

**Reply by Dariusz1989** [reply]
Hello
Thank you for your super fast reply!
Yeh I though I "had it" but looks like I failed... will need to extend my implementation.

**Reply by Dariusz1989** [reply]
Ok
I'm still getting it, this is my config... >
```
    env = os.environ
    sub = subprocess.Popen([
        puthonVirtualEnvironmentPath +"/Scripts/pyinstaller",
        "--clean",
        "--onefile",
        "--disable-windowed-traceback",
        "--noconsole",
        buildAppDir+ "testAppConfig.spec",
        "--distpath",
        buildAppDir+ "dist",
        "--workpath",
        buildAppDir+ "build",
        "--noconfirm",
    ],
        env=env)
```
Is this the wrong place to place it?

**Reply by bwoodsend** [reply]
Oh, you're using the spec file. In which case `--onefile`, `--disable-windowed-traceback` and `--noconsole` are all ignored. Edit `testAppConfig.spec`, find the `exe = EXE(...)` bit and add `disable_windowed_traceback=True` as a parameter to `EXE()` (or just set it to `True` if `disable_windowed_traceback=False` is already there).

**Reply by Dariusz1989** [reply]
> Oh, you're using the spec file. In which case `--onefile`, `--disable-windowed-traceback` and `--noconsole` are all ignored. Edit `testAppConfig.spec`, find the `exe = EXE(...)` bit and add `disable_windowed_traceback=True` as a parameter to `EXE()` (or just set it to `True` if `disable_windowed_traceback=False` is already there).

Hey
Yes that worked, but I still get pop up window & path to exe/etc.
Content is just > 
Traceback is disabled via bootloader option.
How can I entirely disable the window popup?
TIA

**Reply by bwoodsend** [reply]
Hmm, there's no option to get rid of the dialog completely. I guess if you really want a silent exit then just swallow the exception yourself.
```python
try:
    # your code here
except:
    pass
```
